src/Classes/ModList.lua

```diff
@@ -46,7 +46,7 @@ function ModListClass:ReplaceModInternal(mod)
 end
 
 function ModListClass:MergeMod(mod)
-	if mod.type == "BASE" or mod.type == "INC" then
```

Comment:
I felt this was safe because the only places we merge mods are places that don't even have MORE modifiers, let alone multiple.  If we end up being able to scale an item that has 2+ more multipliers of the same type though, we should revisit this to make sure it's accurate